Core/HLE/sceKernelThread.cpp

```diff
@@ -3095,6 +3100,13 @@ void __KernelExecuteMipsCallOnCurrentThread(u32 callId, bool reschedAfter)
 	call->savedV1 = currentMIPS->r[MIPS_REG_V1];
 	call->savedId = cur->currentMipscallId;
 	call->reschedAfter = reschedAfter;
+	
+	// Also need to backup other regs which might be used by the called mips function, needed to prevent corrupting the original regs (ie. Dissidia 012 with more than 2 players, Lord of Arcana)
```

Comment:
I do agree about synced clock, but most games only use timestamp delta from previously recorded timestamp as networking timeout, so synced time will probably not affecting much to the stability.
However the scaled timestamp does affect network stability greatly during my test, especially when each instance is getting different/unstable FPS.